2025-06-15 23:18:28,144 INFO    MainThread:102760 [wandb_setup.py:_flush():81] Current SDK version is 0.20.1
2025-06-15 23:18:28,145 INFO    MainThread:102760 [wandb_setup.py:_flush():81] Configure stats pid to 102760
2025-06-15 23:18:28,145 INFO    MainThread:102760 [wandb_setup.py:_flush():81] Loading settings from /home/jonas-petersen/.config/wandb/settings
2025-06-15 23:18:28,145 INFO    MainThread:102760 [wandb_setup.py:_flush():81] Loading settings from /home/jonas-petersen/dev/Othello_with_RL/wandb/settings
2025-06-15 23:18:28,145 INFO    MainThread:102760 [wandb_setup.py:_flush():81] Loading settings from environment variables
2025-06-15 23:18:28,145 INFO    MainThread:102760 [wandb_init.py:setup_run_log_directory():703] Logging user logs to /home/jonas-petersen/dev/Othello_with_RL/wandb/run-20250615_231828-35i6ec52/logs/debug.log
2025-06-15 23:18:28,145 INFO    MainThread:102760 [wandb_init.py:setup_run_log_directory():704] Logging internal logs to /home/jonas-petersen/dev/Othello_with_RL/wandb/run-20250615_231828-35i6ec52/logs/debug-internal.log
2025-06-15 23:18:28,145 INFO    MainThread:102760 [wandb_init.py:init():831] calling init triggers
2025-06-15 23:18:28,145 INFO    MainThread:102760 [wandb_init.py:init():836] wandb.init called with sweep_config: {}
config: {'episodes': 50, 'epsilon': 0.2, 'lr': 0.001, '_wandb': {}}
2025-06-15 23:18:28,145 INFO    MainThread:102760 [wandb_init.py:init():872] starting backend
2025-06-15 23:18:28,355 INFO    MainThread:102760 [wandb_init.py:init():875] sending inform_init request
2025-06-15 23:18:28,361 INFO    MainThread:102760 [wandb_init.py:init():883] backend started and connected
2025-06-15 23:18:28,363 INFO    MainThread:102760 [wandb_init.py:init():956] updated telemetry
2025-06-15 23:18:28,370 INFO    MainThread:102760 [wandb_init.py:init():980] communicating run to backend with 90.0 second timeout
2025-06-15 23:18:48,052 WARNING MainThread:102760 [wandb_init.py:init():1613] [no run ID] interrupted
Traceback (most recent call last):
  File "/home/jonas-petersen/dev/Othello_with_RL/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1609, in init
    return wi.init(run_settings, run_config, run_printer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jonas-petersen/dev/Othello_with_RL/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 999, in init
    result = wait_with_progress(
             ^^^^^^^^^^^^^^^^^^^
  File "/home/jonas-petersen/dev/Othello_with_RL/.venv/lib/python3.12/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 24, in wait_with_progress
    return wait_all_with_progress(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jonas-petersen/dev/Othello_with_RL/.venv/lib/python3.12/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 87, in wait_all_with_progress
    return asyncio_compat.run(progress_loop_with_timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jonas-petersen/dev/Othello_with_RL/.venv/lib/python3.12/site-packages/wandb/sdk/lib/asyncio_compat.py", line 30, in run
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/concurrent/futures/_base.py", line 451, in result
    self._condition.wait(timeout)
  File "/usr/lib/python3.12/threading.py", line 355, in wait
    waiter.acquire()
KeyboardInterrupt
2025-06-15 23:18:48,377 INFO    MsgRouterThr:102760 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 2 handles.
